resources/views/emails/exposure.blade.php

```diff
@@ -0,0 +1,13 @@
+@component('mail::message')
+# Hello {{ $first_name }},
+
+You are receiving this message because you visited the Student Competition Center {{ $date }} according to the People Counter system.
+Please see below for an important message from Dr. Cunefare.
```

Comment:
If this is intended to be on a separate line, you need to add a blank line above it. I think either way might be fine?

Reply:
Same line is fine, was just trying to avoid a single suuuuper long line in the file.